pkgs/by-name/md/mdt/package.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "mdt";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "henriklovhaug";
+    repo = "md-tui";
+    rev = "v${version}";
+    hash = "sha256-R6HTdNoPMlfC644AhGFfCLPojm3O2DXpmtSy1S9MCz4=";
+  };
+
+  cargoHash = "sha256-GgwC25mrEePAT5PlNypyPo5qy9mC+lxVwOG1YozAicw=";
+
```

Comment:
```suggestion
    buildInputs = lib.optionals stdenv.isDarwin [ darwin.apple_sdk.frameworks.CoreServices ];
```

Reply:
i can't test this, are you sure that it'll work?

Reply:
Yep, tested.

Reply:
Added, thank you for testing.